user-guide/Advanced_Modules/User Defined APIs/Tutorial/UD_APIs_Hello_world_tutorial.md

```diff
@@ -0,0 +1,231 @@
+---
+uid: UD_APIs_Hello_world_tutorial
+---
+
+# Hello world tutorial
+
+This tutorial shows you how to create your first custom API. You will create a simple API that, upon execution, will return "Hello world". The content and screenshots for this tutorial have been created in DataMiner version 10.3.9.
+
+> [!IMPORTANT]
+> Before you start this tutorial, take a look at the [prerequisites](xref:UD_APIs_Tutorials).
+
+## Overview
+
+- [Step 1: Create an Automation script solution](#step-1-create-an-automation-script-solution)
+- [Step 2: Create the Automation script](#step-2-create-the-automation-script)
+- [Step 3: Create an API token](#step-3-create-an-api-token)
+- [Step 4: Create the API Definition](#step-4-create-the-api-definition)
+- [Step 5: Trigger the API using Postman](#step-5-trigger-the-api-using-postman)
+
+## Step 1: Create an Automation script solution
+
+While you could develop the Automation script in Cube, creating an Automation script solution in Visual Studio will give you the advantage of having access to all the features of Visual Studio and [DIS](xref:DIS). If you also use a version control system like Git, this will also enable versioning and make collaboration possible.
+
+1. Create an Automation script solution:
+
+   1. In Visual Studio, select *Create a new project*.
+
+   1. Search for *DataMiner Automation Script Solution (Skyline Communications)* in the template search box and click *Next*.
+
+      ![DataMiner Automation Script Solution in Visual Studio](~/user-guide/images/UDAPIS_Tutorials_HelloWorld_DataMiner_DIS_Solution.jpg)
+
+   1. Use "DataMinerAPIs" as the name for your solution, so you can reuse this solution for any future API scripts.
+
+   1. Choose a location to save the Automation script solution and click *Next*.
+
+      ![Visual Studio configure your project](~/user-guide/images/UDAPIS_Tutorials_HelloWorld_DIS_SolutionSettings.jpg)
+
+## Step 2: Create the Automation script
+
+The logic of your API is in an Automation script. This processes input arguments, executes logic, and returns a response. In this example, no input is processed, and no logic is executed. The API will only return "Hello world!" to the user.
+
+### Preparing the project
+
+You first need to prepare the project so the dependencies are up to date. To develop API scripts, you will need the "Skyline.DataMiner.Dev.Automation" NuGet package of at least version 10.2.12.
```

Comment:
Do you mean DataMiner 10.2.12? Could you clarify this?

Reply:
The 10.2.12 applies to the version of the NuGet package, which is in line with the version of DataMiner. The types needed to build this script have been available since 10.2.12, but it does seem weird to suggest that since this feature should only be used from 10.3.6 onwards. I will update it to 10.3.6.